package.json

```diff
@@ -13,6 +13,12 @@
     "type": "git",
     "url": "git+https://github.com/ExodusMovement/zip-up.git"
   },
+  "files": [
+    ".nvmrc",
+    "bundle.js",
+    "package.json",
+    "README.md"
```

Comment:
`package.json` and `README.md` are implicitly assumed: https://docs.npmjs.com/files/package.json.html#files

Actually, `main` is implcitly assumed as well; I wonder if you could just set an empty array here. Try it, then run `npm pack --dry-run` to see what files will be published.

Reply:
I'll try it with empty. Without the `files` field at all, *all* files get zipped up.

Reply:
Tried it, and it looks like it will work! Submitting new commit.
![image](https://user-images.githubusercontent.com/643129/54160660-c44ea080-4415-11e9-907f-5850a6ded8fa.png)